pandas/plotting/_core.py

```diff
@@ -2128,50 +2128,74 @@ def hist_frame(data, column=None, by=None, grid=True, xlabelsize=None,
                xrot=None, ylabelsize=None, yrot=None, ax=None, sharex=False,
                sharey=False, figsize=None, layout=None, bins=10, **kwds):
     """
-    Draw histogram of the DataFrame's series using matplotlib / pylab.
+    Draw histogram of the DataFrame's Series using matplotlib.
```

Comment:
Perhaps this is written like this in other parts of the docs, but I personally find it easier to understand to say something like `Draw histogram of the DataFrame's columns`, instead of `DataFrame's Series`.